From e73456dfbb36c8d19a802ffe46b8ebccd77ccb22 Mon Sep 17 00:00:00 2001
From: Sami Kerola <kerolasa@iki.fi>
Date: Fri, 8 Feb 2019 22:01:05 +0000
Subject: [PATCH 649/686] include: add no return function attribute

This warning is repeated 112 times when compiling with all warnings.

xalloc.h:23:1: warning: function '__err_oom' could be declared with
attribute 'noreturn' [-Wmissing-noreturn]

Signed-off-by: Sami Kerola <kerolasa@iki.fi>
---
 include/xalloc.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/xalloc.h b/include/xalloc.h
index 1d225d4..0129a85 100644
--- a/include/xalloc.h
+++ b/include/xalloc.h
@@ -19,7 +19,8 @@
 # define XALLOC_EXIT_CODE EXIT_FAILURE
 #endif
 
-static inline void __err_oom(const char *file, unsigned int line)
+static inline void __attribute__((__noreturn__))
+__err_oom(const char *file, unsigned int line)
 {
 	err(XALLOC_EXIT_CODE, "%s: %u: cannot allocate memory", file, line);
 }
-- 
1.8.3.1

